### ISSUE TITLE
Add "disable this filter" button to Logger

### ISSUE THREAD

**Issue by jspenguin2017** [NanoAdblocker/NanoCore#2]
Use `badfilter` for network filters and `#@#` for others.

**Comment by jspenguin2017** [comment]
Blocked by https://github.com/NanoAdblocker/NanoCore/issues/33

**Comment by jspenguin2017** [comment]
Merged into https://github.com/NanoAdblocker/NanoCore/issues/11
